patterns/2-structured/project-setup/communication-tooling.md

```diff
@@ -32,29 +30,28 @@ answer.
+2. **one or more public discussion channels** that come with less rigid a structure. Typically, this will be mailing lists, online forums or even archived chat channels. Usually it is enough to start with just one channel for the project. If traffic increases too much it is helpful to split discussions about project usage from discussions about project development.
+3. **a private channel** where communication about sensitive topics can happen between [Trusted Committers](../trusted-committer.md) - e.g. adding further Trusted Committers to the host team. This channel should be used with great care such that communication defaults to open and is kept private only under very rare circumstances.
 
-- The project should have its own issue tracker where structured communication, decision-making and progress tracking can happen transparently for all host team members but also for downstream users and contributors to follow.
-- The project should have one or more discussion channels that come with less rigid a structure. Typically, this will be mailing lists, online forums or even archived chat channels. Usually it is enough to start with just one channel for the project, if traffic increases too much it's helpful to split discussions about project usage from discussions about project development.
-- In addition, the project should have one private channel where sensitive communication can happen between [Trusted Committers](../trusted-committer.md) - e.g. adding further Trusted Committers to the host team. This channel should be used with great care such that communication defaults to open and is kept private only under very rare circumstances.
+While communication can happen outside of those written channels, as much information as possible should be brought back to the asynchronous channels.
 
-While communication can happen outside of written channels, as much information
-as possible should be brought back to the asynchronous channels.
+All communication channels should be documented in the project `README.md`. For more details on the use of this file see [Standard Base Documentation](./base-documentation.md).
 
-All communication channels should be documented in the project `README.md`. The
-host team members need to make an effort to direct questions that they receive
-personally back to official communication channels.
+The host team members need to make an effort to direct questions that they receive personally (e.g. via email or private chat messages) back to official communication channels.
 
 ## Resulting Context
 
 Setting up and consistently using official asynchronous communication channels
-helps create a base level of passive documentation that can be referenced again
-when similar questions come up again.
+helps create a base level of [passive documentation](https://www.oreilly.com/library/view/understanding-the-innersource/9781491986899/ch04.html) that can be referenced again when similar questions come up again.
```

Comment:
@MaineC I heard you using the term "passive documentation" regularly.
Do you have any reference/explanation that you particularly like?

So far I have linked to one of the ISC resources, which could be some nice Marketing too :)

Reply:
Q&A systems are another alternative for communications that creates passive documentation, for example some companies have things analogous to StackOverflow internally. Might be worth mentioning them.

Reply:
> Q&A systems are another alternative for communications that creates passive documentation, for example some companies have things analogous to StackOverflow internally. Might be worth mentioning them.

Answering myself, this is probably covered in "online forums" mentioned earlier anyway.

Reply:
Interesting point. Yeah, StackOverflow-like systems could be considered a special form of online forums. However calling this out explicitly could be helpful as e.g. I did not think about those either when reading this pattern.

Reply:
Added 'Q&A systems' as a possible public communication channel.
Thanks for the suggestions @ranton256 !